.github/workflows/add-label-when-promoted.yaml

```diff
@@ -0,0 +1,35 @@
+name: Check if commits are promoted
+
+on:
+  push:
+    branches:
+      - master
+      - branch-*.*
```

Comment:
We have `manager-3.2` branch and `perf-v14`.
Shouldn't those also be supported here?

Reply:
should add:

- branch-perf-v*
- manager-*.*

Reply:
and later we should add those to mergify configuration

Reply:
Added